### PR TITLE
chore (docs): governance changes

### PR DESCRIPTION
**Pull request by avishnu**
Few typos and fixes to the Governance document.

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -30,53 +30,52 @@ The OpenEBS project has four core roles. All project members operate in one (or
...
+    * Participate in discussions, contributions, code and documentation reviews for three months or more
+    * Perform reviews of and contribute to non-trivial pull requests
+    * Demonstrate alignment with the project vision and roadmap
+* ability to write quality code and/or documentation
+* ability to collaborate with the team
+* understanding of how the team works (policies, processes for testing and code review, etc)
+* understanding of the project's code base and coding and documentation style
 
-A new Maintainer is nominated by an existing maintainer, a public vote is undertaken, and a 50% vote of existing Maintainers approves the application. A new maintainer’s nomination will be evaluated without prejudice to employer or demographics.
+A new Maintainer is nominated by an existing maintainer, a public vote is undertaken, and a 50% vote of existing Maintainers (rounded off to the ceiling) approves the application. A new maintainer’s nomination will be evaluated without prejudice to employer or demographics.
```
Comment (avishnu) [inline GOVERNANCE.md:67]: For eg. if there are 5 maintainers, at least 3 should vote in favor.

File: GOVERNANCE.md [hunk 2]
```diff
@@ -30,53 +30,52 @@ The OpenEBS project has four core roles. All project members operate in one (or
...
 
 Maintainers who are selected will be granted necessary GitHub rights, and invited to the private maintainer mailing list.
 
 
 #### Removing a Maintainer
 Maintainers may resign at any time if they feel that they will not be able to continue fulfilling their project duties. Maintainers may also be removed after being inactive, failure to fulfill their Maintainer responsibilities, violating the Code of Conduct, or other reasons.
 
-A Maintainer may be removed at any time by a ⅔ vote of the remaining maintainers.
+A Maintainer may be removed at any time by a ⅔ vote of the remaining maintainers, rounded off to the floor.
```
Comment (avishnu) [inline GOVERNANCE.md]: If there are 5 maintainers, at least 3 should vote in favor of the removal

File: GOVERNANCE.md [hunk 3]
```diff
@@ -30,53 +30,52 @@ The OpenEBS project has four core roles. All project members operate in one (or
...
+A new Maintainer is nominated by an existing maintainer, a public vote is undertaken, and a 50% vote of existing Maintainers (rounded off to the ceiling) approves the application. A new maintainer’s nomination will be evaluated without prejudice to employer or demographics.
 
 Maintainers who are selected will be granted necessary GitHub rights, and invited to the private maintainer mailing list.
 
 
 #### Removing a Maintainer
 Maintainers may resign at any time if they feel that they will not be able to continue fulfilling their project duties. Maintainers may also be removed after being inactive, failure to fulfill their Maintainer responsibilities, violating the Code of Conduct, or other reasons.
 
-A Maintainer may be removed at any time by a ⅔ vote of the remaining maintainers.
+A Maintainer may be removed at any time by a ⅔ vote of the remaining maintainers, rounded off to the ceiling.
```
Comment (avishnu) [inline GOVERNANCE.md:75]: For eg. if there are 5 maintainers, at least 3 should vote in favor of the removal.

**Review by orville-wright — APPROVED** [review]
LGTM

**Pull request merged**